### ISSUE TITLE
connection drop after resizing window

### ISSUE THREAD

**Issue by brl0** [yuvipanda/kubessh#18]
Connection drops after resizing the terminal window.  This occurs for me on Windows using multiple terminal emulators, and for @t-potts on Ubuntu.

**Comment by t-potts** [comment]
@yuvipanda Any ideas on how to approach this?

**Comment by mvoelske** [comment]
Can confirm. Here's the relevant portion from my server log:

```
[asyncssh] [conn=112041, chan=0] Recceived window change: 232x50
[asyncssh] [conn=112041, chan=0] Uncaught exception
Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/kubessh/pod.py", line 261, in execute
    await future
  File "/usr/local/lib/python3.8/site-packages/asyncssh/stream.py", line 105, in read
    return await self._session.read(n, self._datatype, exact=False)
  File "/usr/local/lib/python3.8/site-packages/asyncssh/stream.py", line 469, in read
    raise exc
asyncssh.misc.TerminalSizeChanged: Terminal size change: (232, 50, 0, 0)

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/lib/python3.8/site-packages/asyncssh/connection.py", line 561, in _reap_task
    task.result()
  File "/usr/local/lib/python3.8/site-packages/kubessh/app.py", line 98, in handle_client
    await pod.execute(process)
  File "/usr/local/lib/python3.8/site-packages/kubessh/pod.py", line 262, in execute
    except asyncssh.misc.TerminalSizeChanged as exc:
NameError: name 'asyncssh' is not defined
```

**Comment by yuvipanda** [comment]
Fixed by #21, thanks to @mvoelske!
